lib/tls.js

```diff
@@ -140,7 +141,7 @@ function unfqdn(host) {
 function splitHost(host) {
   // String#toLowerCase() is locale-sensitive so we use
   // a conservative version that only lowercases A-Z.
-  const replacer = (c) => String.fromCharCode(32 + c.charCodeAt(0));
+  const replacer = (c) => StringFromCharCode(32 + c.charCodeAt(0));
```

Comment:
```suggestion
  const replacer = (c) => StringFromCharCode(32 + StringPrototypeCharCodeAt(c, 0));
```
the line below this also uses replace/split